app/controllers/patients_controller.rb

```diff
@@ -72,7 +72,6 @@ def update
     else
       error = @patient.errors.full_messages.to_sentence
       flash.now[:alert] = error
-      response.status = :not_acceptable
```

Comment:
this (and the few similar changes) are, I think, because UJS had an update such that it doesn't fire templates on 4XX response codes. So we (unfortunately) treat failures to save as 200s now. it is horrible but we aren't interacting with this through, like, a json api so I think it's okay. I promise I feel an appropriate amount of bad aboutit.